ts/src/coinbase.ts

```diff
@@ -21,7 +21,12 @@ export default class coinbase extends Exchange {
             'name': 'Coinbase',
             'countries': [ 'US' ],
             'pro': true,
-            'rateLimit': 400, // 10k calls per hour
+            // rate-limits:
+            // ADVANCED API: https://docs.cloud.coinbase.com/advanced-trade-api/docs/rest-api-rate-limits
+            // - max 30 req/second for private data, 10 req/s for public data
+            // DATA API    : https://docs.cloud.coinbase.com/sign-in-with-coinbase/docs/rate-limiting
+            // - max 10000 req/hour (to prevent userland mistakes we apply ~3 req/second RL per call
+            'rateLimit': 10,
```

Comment:
I don't see any weight = 1, should we have this rate limit so low?

Reply:
i was thinking have small rl because even though we don't have `1` weight assigned to endpoints, users could use implicit calls in "bucket" like approach, where bucket-like calls can be made in expense of roundtrip hardcaps. 
however, i'll modify per your saying, i think i shouldn't have done that at this stage, and in future i'll add buckets in coinbase.

Reply:
updated.